src/app/public/modules/progress-indicator/progress-indicator.component.ts

```diff
@@ -114,8 +114,13 @@ export class SkyProgressIndicatorComponent implements AfterContentInit, OnDestro
         firstItem.isActive = true;
       }
     }
-    this.progressChanges.emit({
-      activeIndex: this.activeIndex
+    
+    // setTimeout required to allow other ngAfterViewContentInit's of
```

Comment:
Thanks for documenting this!

Is there anyway to write some coverage around this issue?

Reply:
I'm not sure how to unit test this properly, but I think I can make a visual test

Reply:
If you use `fakeAsync`, you can spy on the `progressChanges` emit method.

Reply:
test added